.eslintrc.js

```diff
@@ -1,280 +1,214 @@
 module.exports = {
-    'env': {
-        'browser': true,
-        'commonjs': true,
-        'es6': true
+    env: {
+        browser: true,
```

Comment:
I'm just trying to let my OCD go to my tooling. Of course, then my tooling OCD becomes a problem...